src/pages/products/sgid/environment/air-emissions-inventory.astro

```diff
@@ -0,0 +1,80 @@
+---
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/products/sgid/types';
+import { ProductType, SgidCategory } from '@models/products/sgid/types';
+
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+import UpdateHistory from '@components/data/UpdateHistory.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import Contacts from '@components/page/Contacts.astro';
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah DAQ Air Emissions Inventory',
+  description: 'Facility locations and annual emissions (in tons) for pollutants in Utah.',
+  stewards: ['DEQ DAQ'],
```

Comment:
**question**: are these separate or the same?

Reply:
DAQ is a division of DEQ. So it looked weird to me to see "DEQ & DAQ" so I combined them into a single string. I'm open to other ideas.

Reply:
DAQ is a division of DEQ. So it looked weird to me to see "DEQ & DAQ" so I combined them into a single string. I'm open to other ideas.